ToBeRenamed/Factories/SqlConnectionFactory.cs

```diff
@@ -13,7 +13,7 @@ public SqlConnectionFactory(IConfiguration configuration)
             _connectionString = configuration["ConnectionStrings:DefaultConnection"];
         }
 
-        public IDbConnection GetSqlConnection()
+        public DbConnection GetSqlConnection()
```

Comment:
For some reason, the Respawn library needed this to be a DbConnection instead of an IDbConnection. While Respawn doesn't use this factory, it uses the test factory, and they both implement the same ISqlConnectionFactory.